edge/pkg/edged/edged.go

```diff
@@ -179,6 +179,8 @@ type edged struct {
 	namespace                 string
 	nodeName                  string
 	runtimeCache              kubecontainer.RuntimeCache
+	customInterfaceName       string
+	customInterfaceEnabled    bool //siredmar
```

Comment:
If `customInterfaceName` is not empty, it will take effect. There is no need for `customInterfaceEnabled` parameter

Reply:
you are right, i will remove that enable flag